src/templates/blog-index.tsx

```diff
@@ -117,3 +116,6 @@ export default function BlogIndex({ posts }: BlogIndexProps) {
     </Layout>
   )
 }
+export const Head = () => {
```

Comment:
This component still has `<Layout title="Blog">` (and probably other pages)

Since the Head component is not used in the layout anymore the props should be removed

Reply:
Good catch!